rskj-core/src/test/java/co/rsk/util/MaxSizeHashMapTest.java

```diff
@@ -1,36 +1,39 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2019 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
 package co.rsk.util;
 
-import org.ethereum.TestUtils;
 import org.junit.jupiter.api.Test;
 
 import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
 
 class MaxSizeHashMapTest {
 
     @Test
-    void maxSizeMap_Test() {
-        int maxSize = 50_000;
+    void maxSizeMapTest() {
```

Comment:
why was this test updated?

Reply:
there are couple of reasons: first, this test tries to "test" the feature not as a "blackbox", but an internal state of the instance of a system class. Secondly and mosts importantly, this test uses reflection to obtain an internal field of the system class (via `field.setAccessible(true)`), which is prohibited for non-open java packages in recent versions of junit. There's a workaround by opening system package (via `--add-opens java.base/java.util=ALL-UNNAMED`), but I'm not sure that worth it